src/allmydata/web/directory.py

```diff
@@ -957,18 +1101,19 @@ def __init__(self, client, monitor):
     # The default format is HTML but the HTML renderer is just renderHTTP.
     render_HTML = None
 
-    def slashify_path(self, path):
-        if not path:
-            return ""
-        return "/".join([p.encode("utf-8") for p in path])
+    def render_HTML(self, req):
```

Comment:
There is a `render_HTML` above which I suppose is now obsolete?  I don't see how it could be doing anything, given this new `render_HTML` method, anyway.